core/payment/src/schema.rs

```diff
@@ -87,6 +87,17 @@ table! {
     }
 }
 
+table! {
```

Comment:
table? same macro for views?

Reply:
Yep, there's no separate macro for views. Also, as far as selects are concerned, there isn't much difference between these two.